lib/markdownToAnnotated.ts

```diff
@@ -1,17 +1,6 @@
 import * as builder from "annotatedtext-remark";
 
 const builderOptions = builder.defaults;
-const original_interpretmarkup = builder.defaults.interpretmarkup;
```

Comment:
This code was originally to try to catch spelling errors within backticks, but that proved way more noisy (as it's usually variable names and other code or shell elements within the backticks).